app/javascript/lib/schnelleinsteig_legacy_tests.js

```diff
@@ -1,1847 +0,0 @@
-// This file is not run anymore because it is out of date and not worth
```

Comment:
Sad to see this go, but it's ok! 😢 Thanks for removing it!

Reply:
yeah I don't think these get run in CI or anything, do they? but they are def gonna fail at this point.